pkg/clients/rds/rds.go

```diff
@@ -21,6 +21,8 @@ import (
 	"errors"
 	"time"
 
+	sdkErrors "github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
```

Comment:
Nit: Could this be `sdkerrors`? While camel case is common in Go, I rarely see it in package names (and thus import names). See https://blog.golang.org/package-names for a little more context.

Reply:
Thanks. Already rename the package name.